src/win32-private.c

```diff
@@ -40,7 +40,7 @@ FILE *CreateTempFile (char *filename)
 		return NULL;
 	}
 
-	return fopen (filename, "w");
+	return fopen (filename, "wb");
```

Comment:
there's also this in font.c which I assume should be fixed too:
https://github.com/mono/libgdiplus/blob/d9daa4c86b115f16b5e0a91f912a13baf5d1bbd5/src/font.c#L173